src/v2/Apps/Order/Routes/Counter/index.tsx

```diff
@@ -184,6 +185,7 @@ export class CounterRoute extends Component<CounterProps> {
                   <ShippingSummaryItem order={order} locked />
                   <CreditCardSummaryItem order={order} locked />
                 </Flex>
+                <BuyerGuarantee />
```

Comment:
Nice! Looks like this handles mobile case automatically?

Reply:
oops forgot to add `xs` screenshots. Yes the layout automatically supports small screens.